sunpy/map/mapbase.py

```diff
@@ -529,7 +529,7 @@ def _get_lon_lat(self, frame):
     @property
     def quantity(self):
         """Unitful representation of the map data."""
-        return u.Quantity(self.data, self.unit, copy=False)
+        return u.Quantity(self.data, self.unit, copy=True)
```

Comment:
Probs dont want this but lmao

Reply:
Now loads of this has been fixed can this be reverted?

Reply:
yeah I should do that.

Reply:
Doesn't seem to work:

```
>       value = np.array(
            value, dtype=dtype, copy=copy, order=order, subok=True, ndmin=ndmin
        )
E       ValueError: Unable to avoid copy while creating an array as requested.
E       If using `np.array(obj, copy=False)` replace it with `np.asarray(obj)` to allow a copy when needed (no behavior change in NumPy 1.x).
E       For more details, see https://numpy.org/devdocs/numpy_2_0_migration_guide.html#adapting-to-changes-in-the-copy-keyword.

../../.tox/py312-devdeps/lib/python3.12/site-packages/astropy/units/quantity.py:544: ValueError
```

Not sure if this needs to be patched more in astropy?

Reply:
I believe the `copy` change was to follow `numpy`s lead. See:
https://github.com/astropy/astropy/issues/16167
Several `astropy` downstream packages have received updates to accommodate this change like the following:
https://github.com/astropy/specutils/commit/5ec4931ff66454b545ccf45013b03b1265b97709

Reply:
Thanks for the context, I can follow specutils.